lib/recurly/Pager.js

```diff
@@ -51,7 +51,7 @@ class Pager {
   async first () {
```

Comment:
If you were to iterate over this pager after calling `first`, I believe it could result in iterating 1 item at a time. What do you think about this to get rid of the statefulness:

```js
let params = Object.assign({}, this.params)
params.limit = 1
let results = await this.client._makeRequest('GET', this.path, null, { params: params })
```